Doc/tools/.nitignore

```diff
@@ -32,12 +27,9 @@ Doc/c-api/set.rst
 Doc/c-api/stable.rst
 Doc/c-api/structures.rst
 Doc/c-api/sys.rst
-Doc/c-api/tuple.rst
```

Comment:
This file wasn't modified so the inline check annotations aren't shown, but it still contains one warning, which (given it was removed from `.nitignore`) is now failing CI:

```
Warning: /home/runner/work/cpython/cpython/Doc/c-api/tuple.rst:166: WARNING: py:attr reference target not found: fields
```

The [relevant line](https://github.com/python/cpython/blob/90675b6538476b3eae4fe9f909079785112ba960/Doc/c-api/tuple.rst?plain=1#L168):

```rst
   :attr:`fields` array of the :c:type:`PyStructSequence_Desc` determines which
```

`fields` is a member of the [`PyStructSequence_Desc` struct]() (and part of the limited API/stable ABI), so it should be cross-referencable with `` :c:member:`~PyStructSequence_Desc.fields` ``.

However, while the struct is [properly documented](https://github.com/python/cpython/blob/90675b6538476b3eae4fe9f909079785112ba960/Doc/c-api/tuple.rst?plain=1#L144) (a few lines above in the same file), the members aren't, and are instead just manually listed in an ad-hoc table instead of just using the actual `c:member` directive, i.e.

```rst
.. c:type:: PyStructSequence_Desc
   
   .. c:member:: PyStructSequence_Field *fields

      Pointer to ``NULL``-terminated array with field names of the new type.
```

So, it is a valid warning, and should of course not be simply ignored. For now I suggest fixing the cross-reference to the correct `` :c:member:`~PyStructSequence_Desc.fields` `` (so it is correct, and works once the member is documented), and either:

* Using the `c:member` directive for this and the other 5 members in the file so it is correctly documented and fix the underlying cause of the warning (I can commit a fix to your branch if you like, since its a basically mechanical task), or
* Removing this file from `.nitignore` and we can fix it in a followup

Reply:
See #107316.